sdk/services/prices.ts

```diff
@@ -219,6 +201,58 @@ export default class PricesService {
 		return offChainPrices;
 	}
 
+	private connectToPyth(networkId: NetworkId, server: PriceServer) {
+		this.pyth = new EvmPriceServiceConnection(getPythNetworkUrl(networkId, server), {
+			logger: LOG_WS ? console : undefined,
+		});
+
+		this.pyth.onWsError = (error) => this.retryConnection(networkId, error);
+
+		this.sdk.context.events.emit('prices_connection_update', {
+			connected: true,
+		});
+
+		return this.pyth;
```

Comment:
As this is private and the return object never used I think we can remove this return line

Reply:
Yes. That's correct. Updated